Misc/python.man

```diff
@@ -502,6 +502,9 @@ show how long each import takes. This is exactly equivalent to setting
 .IP PYTHONBREAKPOINT
 If this environment variable is set to 0, it disables the default debugger. It
 can be set to the callable of your debugger of choice.
+.IP PYTHONHISTORY
+This environment variable can be used to set the location of a history file
+(it is \fI~/.python_history\fP by default).
```

Comment:
Add the windows default as well.

Reply:
To be frank, I'm not sure if the Windows default should be mentioned in the man page...

Reply:
There is nothing wrong with mentioning Windows ops in `man` pages.